src/Associations.cc

```diff
@@ -204,12 +214,15 @@ void Associations::collectRefStars(afw::table::SimpleCatalog &refCat, afw::geom:
         double dec = lsst::afw::geom::radToDeg(coord.getLatitude());
         auto star = std::make_shared<RefStar>(ra, dec, flux, fluxErr);
 
-        // TODO DM-10826: RefCats aren't guaranteed to have position errors.
-        // TODO: Need to devise a way to check whether the refCat has position errors
-        // TODO: and use them instead, if available.
-        // cook up errors: 100 mas per cooordinate
-        star->vx = std::pow(0.1 / 3600 / cos(coord.getLatitude()), 2);
-        star->vy = std::pow(0.1 / 3600, 2);
+        if (isnan(refCoordinateErr)) {
```

Comment:
So, if this optional config refCoordinateErr is set, it always uses it?

Reply:
Yes. I've clarified the docstring on `collectRefStars` spells that out, and I've clarified the docstring for the config option to make it more clear.